index.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+import { PvRecorder } from "@picovoice/pvrecorder-node";
+import { selectPrompt } from "./prompt.js";
+import { playVisualizer } from "./visualizer.js";
+import { runCalibration } from "./calibration.js";
+
+const DEFAULT_SAMPLE = [200, 3000];
+
+async function main() {
+  const availableMicrophones = PvRecorder.getAvailableDevices();
+  if (availableMicrophones[0] === "NULL Capture Device") {
+    console.log("Sorry, microphone was not detected.");
+
+    return;
+  }
+
+  console.clear();
+
+  const select = await selectPrompt(
+    "Choose 'Sound Calibration' to set up the visualizer for your mic, or 'Skip' to just start!",
+    [
+      { name: "Sound Calibration", value: "calibrate" },
+      { name: "Skip (Use Defaults)", value: "skip" },
+    ],
+  ).run();
+
+  if (select === "skip") {
+    return await playVisualizer(DEFAULT_SAMPLE);
+  }
+
+  console.clear();
+
+  let volumeSamples = await runCalibration();
+
+  const lowest = volumeSamples[0];
+  const highest = volumeSamples[1];
+  if (lowest > highest || highest - lowest < 800) {
+    volumeSamples = DEFAULT_SAMPLE;
+  }
+
+  await playVisualizer(volumeSamples);
```

Comment:
```suggestion
let volumeSamples = DEFAULT_SAMPLE;
if (select === "calibrate") {
  // キャリブレーションの結果を volumeSamples にセット
}
await playVisualizer(volumeSamples);
```

途中でreturnしない方が素直に上から下に読めるコードになりそうです

Reply:
修正しました！